scripts/runfabtests.sh

```diff
@@ -284,11 +290,60 @@ function cs_test {
 	fi
 }
 
+function complex_test {
+	local test=$1
+	local config=$2
+	local test_exe="fi_${test}"
+	local start_time
+	local end_time
+	local test_time
+
+	# TODO: Add a better way to get to $datarootdir
+	if [ ! -f "${BIN_PATH}../share/fabtests/test_configs/${PROV}/${config}.json" ]; then
```

Comment:
I was suggesting modifying ubertest to return ENODATA or similar to indicate that the test failed as not run.